.github/workflows/scan-controller-image.yaml

```diff
@@ -0,0 +1,68 @@
+name: Scan controller image
+on:
+  workflow_dispatch:
+  # no content, allows manual triggering
+
+  schedule:
+    # 5:30 pm every Sunday (UTC)
+    # to pick up any important bug fixes etc in base image
+    - cron: '30 17 * * 0'
+
+  push:
+    # run when changes pushed to go.mod on main branch
+    branches:
+      - main
+    paths:
+      - ./v2/go.mod
+
+jobs:
+  deploy-site:
```

Comment:
Is this the right name for this job?

Reply:
Good catch! Thanks. I copied some of this from other workflow and might've hit undo somewhere.

Reply:
Making it to be called `scan-image`